src/ethereum_test_tools/rpc/rpc.py

```diff
@@ -6,6 +6,7 @@
 from typing import Any, Dict, List, Literal, Optional, Union
 
 import requests
+from cli.gentest import RequestManager
```

Comment:
This introduces a circular import. I think we should avoid using `RequestManager` objects here in rpc.py. 

The easiest way to resolve this would be to create a dict of the required arguments in `gentest` and have `debug_trace_call` take this dictionary as an argument. 

We could consider defining a small pydantic BaseModel of the required arguments and passing an instance of this model instead for better type-checking of the parameters, but I'm not sure it's worth it here.